internal/commands/scan.go

```diff
@@ -502,7 +502,8 @@ func scanCreateSubCommand(
 		"",
 		fmt.Sprintf("Parameters to use in SCA resolver (requires --%s).", commonParams.ScaResolverFlag),
 	)
-	createScanCmd.PersistentFlags().String(commonParams.ScanTypes, "", "Scan types, ex: (sast,iac-security,sca,api-security)")
+	createScanCmd.PersistentFlags().String(commonParams.ContainerImagesFlag, "", "List of container images to scan, ex: manuelbcd/vulnapp:latest,debian:10")
```

Comment:
Are we using ex as the example in more places?
I think it is better to write the full word if not / keep the same as we do

Reply:
Yes, we use ex in all of our flag description